include/glow/Graph/Nodes.h

```diff
@@ -355,6 +356,15 @@ class NodeVisitor : public NodeVisitorBase<ImplClass> {
 #include "glow/AutoGenNodes.def"
 };
 
+/// Signifiers for exporting and importing properties of Nodes.
+constexpr char layoutSignifier[] = "layout";
```

Comment:
Nit: Does it really need to be initialized in the header?

Reply:
It's a `constexpr` so it must be immediately initialized. And it's used in both Importer and Exporter libs so needs to live in a header.